lib/klp.pm

```diff
@@ -45,14 +46,23 @@ sub install_klp_product {
     if ($livepatch_repo) {
         zypper_ar("$utils::OPENQA_FTP_URL/$livepatch_repo", name => "repo-live-patching");
     }
-    else {
+    elsif (!is_sle_micro) {
```

Comment:
Does it mean this branch is for sle and something else?

Reply:
Yes, it is for sle and we don't need to do anything for micro.

Reply:
IMO It would be more precise to have the condition adapted to sle only.

Reply:
Yeah,  it was obvious, it should be fixed with self explanatory logic.

Reply:
I am out of the loop, so initially it looked like live patching might work for leap micro, too

Reply:
This is specifically about SL Micro 6.0 and SUSE maintenance, not Leap Micro.